src/05-led-roulette/my-solution.md

```diff
@@ -75,7 +76,8 @@ section               size        addr
 .debug_aranges         160         0x0
 Total               747766
 
-target/thumbv7em-none-eabihf/release/led-roulette  :
+$ cargo size --bin led-roulette -- -A
+led-roulette  :
```

Comment:
This is exactly the same command as above and shouldn't lead to different results (release vs debug size).
It looks like the 'release'-parameter is missing.

So I used nearly same as above but with added 'release' tag to get a result.
`cargo size --bin led-roulette --target thumbv7em-none-eabihf --release -- -A`